sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -393,6 +395,54 @@ def test_global_fanout(self):
           | beam.CombineGlobally(combine.MeanCombineFn()).with_fanout(11))
       assert_that(result, equal_to([49.5]))
 
+  @attr('ValidatesRunner')
+  def test_accumulating_combine(self):
```

Comment:
This seems mostly redundant with https://github.com/apache/beam/blob/master/sdks/python/apache_beam/runners/portability/fn_api_runner_test.py#L562 , other than the fact that it does globally as well. (Globally is just built on top of per-key, so there's little value in making it validates runner.)

Reply:
Removed @attr('ValidatesRunner').
Done.